packages/table/src/table.tsx

```diff
@@ -647,7 +646,9 @@ export class Table extends AbstractComponent<ITableProps, ITableState> {
         const correctedScrollTop = this.shouldDisableVerticalScroll() ? 0 : scrollTop;
 
         // defer to the quadrant stack to keep all quadrant positions in sync
-        this.quadrantStackInstance.scrollToPosition(correctedScrollLeft, correctedScrollTop);
+        if (this.quadrantStackInstance != null) {
```

Comment:
No need. This will always exist.

Reply:
can you explain why? 
the classic pitfall with refs is accessing them in a callback after the component has unmounted. can we guarantee that will never be the case?

Reply:
> after the component has unmounted

That's fair. My reasoning was that a `<Table>` will always have both a `<TableQuadrantStack>` (which will in turn always include a `scrollContainerElement` from the `MAIN` quadrant, which will always exist) and a `<ColumnHeader>`.